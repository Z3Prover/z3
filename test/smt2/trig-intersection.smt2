;; Copyright (c) 2016 Microsoft Corporation

(set-info :source |Written by D. B. Staple for GitHub issue #680.|)
(set-info :status sat)

(declare-fun theta () Real)
(assert (= (sin theta) (cos theta)))
(check-sat-using qfnra-nlsat)
(get-model)
