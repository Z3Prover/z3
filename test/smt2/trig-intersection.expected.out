sat
(model 
  (define-fun theta () Real
    (+ (acos (root-obj (+ (* 2 (^ x 2)) (- 1)) 2)) pi))
)
